src/user-icon/user-icon.scss

```diff
@@ -52,6 +52,7 @@
   }
 
   > .iui-icon {
+    display: flex;
```

Comment:
THis change of applying flex everywhere looks weird.. I do see it fixes position, but do you exactly know why?